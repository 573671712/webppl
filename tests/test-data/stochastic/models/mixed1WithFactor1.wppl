var model = function() {
  var x = beta(1, 2);
  var y = flip(x);
  factor(y ? 0 : -2);
  return y;
};
