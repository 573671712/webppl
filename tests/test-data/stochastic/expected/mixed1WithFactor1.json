{
  "hist": {
    "true": 0.786986,
    "false": 0.213014
  }
}
